code/game/machinery/cryopod.dm

```diff
@@ -269,10 +269,10 @@
 					reshape.Crop(7,4,26,31)
 					reshape.Crop(-5,-3,26,30)
 					GLOB.sac_image = reshape
-
-					for(var/datum/mind/H in SSticker.mode.cult)
-						if(H.current)
-							to_chat(H.current, "<span class='danger'>Nar'Sie</span> murmurs, <span class='cultlarge'>[occupant] is beyond your reach. Sacrifice [GLOB.sac_mind.current] instead...</span></span>")
+					if(GLOB.sac_mind == occupant)
```

Comment:
One, GLOB.sac_mind is a mind datum, occupant is a mob. These two will never be equal.

Two, even if they were of the same type, this statement happens after we pick a new target to be the sac_mind and the list of potential targets excludes the occupant, so the occupant could never be selected anyhow.

Three, in the current position even if the GLOB.sac_mind was not updated to the new target yet, all this would do is stop the new target from displaying.

Reply:
Move this up between lines 246 and 247, replace occupant with occupant.mind